controller/contactsController.js

```diff
@@ -86,6 +81,32 @@ router.put("/:contactId", async (req, res, next) => {
   } catch (error) {
     next(error);
   }
-});
+};
+const updateFavorite = async (req, res, next) => {
+  try {
+    const contactId = req.params.contactId;
+    const { favorite } = req.body;
+    const { error } = favoriteFieldSchema.validate({ favorite });
+    if (error) {
+      handleJoiError(error, res);
+    } else {
```

Comment:
```suggestion
    if (error) return  handleJoiError(error, res);
```

Reply:
Kiedy zrobisz early return to unikniesz robienia if else. Kod wtedy jest łatwiejszy do czytania. Zobacz czy to nie ma więcej sensu :)